src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedByDotComa = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.length > 0);
+
+  const splittedByDoubleDot = splittedByDotComa
```

Comment:
The variable name 'splittedByDoubleDot' does not clearly describe its contents. A more descriptive name such as 'styleProperties' would better indicate that it contains an array of style properties split by colons.